src/scroll/scroll_menu.py

```diff
@@ -39,15 +39,25 @@ def set_cancel() -> None:
             """Cancel don't open file"""
             self.future.set_result(None)
 
+        # Raise exception if attempt to open any extension besides .txt or .md
         def set_done() -> None:
             """Future object when done return None"""
```

Comment:
```suggestion
            """Handles actions related to loading previews of files into the file explorer."""
```
Is this an accurate docstring for this function?

Reply:
I think it should be `handles actions related to adding file's contents to text editor`

Reply:
Oh, the code in `scroll_menu.py` is loading files into the text editor? That seems kind of backwards. Maybe add an issue to refactor? Definitely add that docstring too.